src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -83,19 +83,11 @@ impl LevelCompactionPicker {
         stats: &mut LocalPickerStatistic,
     ) -> Option<CompactionInput> {
         let overlap_strategy = create_overlap_strategy(self.config.compaction_mode());
-        // FIXME(li0k): Just workaround, need to use more reasonable way to limit task size
-        let max_depth = std::cmp::max(
-            (self.config.max_compaction_bytes as f64
-                / self.config.sub_level_max_compaction_bytes as f64) as usize,
-            self.config.level0_sub_level_compact_level_count as usize + 1,
-        );
-
         let non_overlap_sub_level_picker = NonOverlapSubLevelPicker::new(
-            0,
+            self.config.sub_level_max_compaction_bytes,
```

Comment:
Is it risky to set both min_bytes and min depth, when the selected files are small (min bytes is not satisfied)?